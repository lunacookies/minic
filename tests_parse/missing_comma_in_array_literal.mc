func main {
	a := [1 2 3, 4]
}
