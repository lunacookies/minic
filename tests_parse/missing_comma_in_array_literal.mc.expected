func main {
	a := [
		1,
		2,
		3,
		4,
	]
}
tests_parse:20..21: error: missing “,”
tests_parse:22..23: error: missing “,”
